R/fit_model.R

```diff
@@ -75,17 +62,24 @@ fit_model <- function(
         rt = rt,
         gp = gp,
         stan = stan,
-        verbose = interactive()
+        verbose = TRUE,
+        # Dump logs to console to be caught by pipeline's logging instead of
+        # EpiNow2's default through futile.logger
+        logs = EpiNow2::setup_logging(
+          threshold = "INFO",
```

Comment:
@seabbs is something off here? The EpiNow2 logger is emitting `[DEBUG]` level logs which should be below the threshold. Any chance this is an EpiNow2 level issue or do you see something obvious here?

Reply:
Now https://github.com/CDCgov/cfa-epinow2-pipeline/issues/102